README.md

```diff
@@ -544,6 +543,7 @@ Important: During credentials rotation, SIMPHERA will not be available for a sho
 
 | Name | Description |
 |------|-------------|
+| <a name="input_account_id"></a> [account\_id](#input\_account\_id) | The AWS account id used to create resources. |

```

Comment:
We use terraform-docs to generate this part of the readme.

Reply:
BTW: terraform-docs will be executed automatically with precommit hooks and GitHub quality gate in upcoming PR: https://github.com/dspace-group/simphera-reference-architecture-aws/pull/80